### PR TITLE
Refactor templated feature layers onAdd

### PR DESCRIPTION
**Pull request by ben-lu-uw**
Addresses https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/662#discussion_r903823184
Duplicate _pullFeatureFeeds in onAdd and onMoveEnd even though onAdd later calls onMoveEnd thus doing the same thing twice

### CODE REVIEW

File: src/mapml/layers/TemplatedFeaturesLayer.js [hunk 1]
```diff
@@ -45,36 +43,8 @@ export var TemplatedFeaturesLayer =  L.Layer.extend({
...
-            return;
-        }
-      _pullFeatureFeed(this._getfeaturesUrl(), 10)
-        .then(function() { 
-              map.addLayer(features);
-              map.fire('moveend');  // TODO: replace with moveend handler for layer and not entire map
-        })
-        .catch(function (error) { console.log(error);});
+
+      map.fire('moveend');  // TODO: replace with moveend handler for layer and not entire map
```
Comment (prushforth) [inline src/mapml/layers/TemplatedFeaturesLayer.js:47]: What would happen if we replaced this event firing with "this.redraw()"?

Comment (ben-lu-uw) [reply src/mapml/layers/TemplatedFeaturesLayer.js:47]: Seemingly nothing changes, same with calling _onMoveEnd directly however not sure why ` map.fire('moveend');` was needed in the first place so needs a bit more digging

Comment (ben-lu-uw) [reply src/mapml/layers/TemplatedFeaturesLayer.js:47]: Needed to fix some bugs: https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/214

Comment (prushforth) [reply src/mapml/layers/TemplatedFeaturesLayer.js:47]: Seems that the map moveend firing may cover up some issues in other templated layers, so we can put that consideration aside for now.

**Pull request merged**